The config parser

ShairYousuf opened the pull request:
The parser should be able to read a YAML config file, currently, no feature has been made on what it will do after that. Currently, it just prints the settings.

Looking for feedback and what to do next.

ShairYousuf committed:
The config parser

The parser should be able to read a yaml config file, currently no feature has been made on what it will do after that. Currently it just prints the settings.

ShairYousuf commented:
![image](https://user-images.githubusercontent.com/71799869/124362340-11458900-dc56-11eb-87fe-2efdb49ccb67.png)
This was the YAML file I used to test.

Burke-Daniel commented on examples/config/config_parser.py:
```diff
@@ -0,0 +1,31 @@
+import yaml
+import platform
+
+print(platform.system())
```
Probably won't want this in here, as I believe this will get called whenever this module gets imported

ShairYousuf replied:
Hey man I changed it.

Burke-Daniel commented on examples/config/config_parser.py:
```diff
@@ -0,0 +1,31 @@
+        print(device)
+        #function basically returns settings info to where ever it needs to go, currently it is just printing 
+
+config = parser("device_config_test.yaml")
+data=config.file_reader()
+devices=data["devices"]
+found=False
+for device in devices:
+    if device==platform.system():
+        config.settings_return(devices[device]) 
+        found=True
+if found==False:
+    config.settings_return(data["devices"]["default"])
+
```
This is a nice little test, but so this only runs with this module is ran explicity, should put this above:
```
if __name__ == '__main__':
...
```

ShairYousuf replied:
Thanks for pointing that out.

Burke-Daniel commented on examples/config/config_parser.py:
```diff
@@ -0,0 +1,31 @@
+print(platform.system())
+
+class parser:
+    def __init__(self,filepath):
```
Good practice to put spaces between comma and next parameter:
```
 def __init__(self, filepath):
```

ShairYousuf replied:
Thanks!!

Burke-Daniel reviewed (changes requested):
Couple small things, and I'll get you to look at my Device class MR, and then probably make some changes to this based off of how the configuration looks over there

ShairYousuf committed:
Updated the Config parser

This is a small update to the config parser, after Daniel's feedback on the initial one
